pkg/config/config.go

```diff
@@ -27,18 +30,21 @@ func New() Config {
 		PostgresContainerName:   getEnv("POSTGRES_CONTAINER_NAME", "postgresql"),
 		OauthProxyContainerName: getEnv("OAUTH_PROXY_CONTAINER_NAME", "ups-oauth-proxy"),
 
-		UPSImageStreamName:        getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
-		UPSImageStreamTag:         getEnv("UPS_IMAGE_STREAM_TAG", "latest"),
+		UPSImageStreamName: getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
+		UPSImageStreamTag:  getEnv("UPS_IMAGE_STREAM_TAG", "latest"),
+
 		OauthProxyImageStreamName: getEnv("OAUTH_PROXY_IMAGE_STREAM_NAME", "ups-oauth-proxy-imagestream"),
 		OauthProxyImageStreamTag:  getEnv("OAUTH_PROXY_IMAGE_STREAM_TAG", "latest"),
 
 		PostgresImageStreamNamespace: getEnv("POSTGRES_IMAGE_STREAM_NAMESPACE", "openshift"),
 		PostgresImageStreamName:      getEnv("POSTGRES_IMAGE_STREAM_NAME", "postgresql"),
-		PostgresImageStreamTag:       getEnv("POSTGRES_IMAGE_STREAM_TAG", "latest"),
+		PostgresImageStreamTag:       getEnv("POSTGRES_IMAGE_STREAM_TAG", "9.6"),
 
 		// these are used when the image stream does not exist and created for the first time by the operator
 		UPSImageStreamInitialImage:        getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/aerogear/unifiedpush-wildfly-plain:2.2.1.Final"),
 		OauthProxyImageStreamInitialImage: getEnv("OAUTH_PROXY_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/openshift/oauth-proxy:v1.1.0"),
+
+		BackupImage: getEnv("BACKUP_IMAGE", "quay.io/integreatly/backup-container:1.0.8"),
```

Comment:
Here should be the latest as it is done in other places.

Reply:
We had an earlier discussion here somewhere about that, and decided to pin it to a specific version, to avoid surprises.

Reply:
@grdryn it is fine. I spoke with Leo and changed it for the fixed version too. 
I just think that it should be done in the CR. See that:

RHMI can setup it now with 1.0.8
But tomorrow we can have a MobileX which will require the version 2.0.0
Each platform will have one release date and process.

So, the appropriate place for any configuration regards the CRD is in its CR. The operator Env Var
should have things that are relevant to the operator itself. For example, the namespaces which can be used for it.

Reply:
Anyway, we can improve it after so I am closing it. I just would like to share the idea with you.